src/lighteval/tasks/registry.py

```diff
@@ -221,7 +255,13 @@ def taskinfo_selector(
     if "." in tasks and os.path.exists(tasks):
         tasks = ",".join([line for line in open(tasks, "r").read().splitlines() if not line.startswith("#")])
 
-    for task in tasks.split(","):
+    # First we unwrap all task groups in tasks (we have to do split after unwrapping as the groups can contain multiple tasks)
+    tasks_list = [
+        task
+        for maybe_task_group in tasks.split(",")
+        for task in task_registry.task_groups_dict.get(maybe_task_group, maybe_task_group).split(",")
+    ]
```

Comment:
Expand to make more readable - I know list comprehensions are more efficient but this snippet is not easy to understand fast.
Some things which must be made clearer:
- why we use `maybe_task_group` as default value if it fails as key
- why it's called `maybe_task_group`